templates/esm/index.html

```diff
@@ -136,28 +134,25 @@ <h4 class="u-no-max-width">Which hardware platforms are supported under Ubuntu E
 
       <h4 class="u-no-max-width">How long will Ubuntu ESM be maintained?</h4>
       <p>This depends on the release. Ubuntu 14.04 (Trusty Tahr) and 16.04 (Xenial Xerus) will have updates provided for up to three years from the <a href="/about/release-cycle">release end-of-life date</a>. Ubuntu 18.04 (Bionic Beaver) and subsequent releases until further announcement will have updates provided for up to five years from the release end-of-life date.</p>
 
       <h4 class="u-no-max-width">Is it possible to purchase Ubuntu ESM months down the road when needed, with or without backdating the cost, or does it need to be in place in advance?</h4>
       <p>You can purchase Ubuntu Advantage support at any time. It does not need to be in place in advance, although we strongly recommend you eliminate the gap between when Ubuntu ESM is enabled on your system(s), to avoid exposing your systems to security vulnerabilities. Ubuntu Advantage is priced year-over-year so there is no backdating.</p>
 
-      <h4 class="u-no-max-width">Any plans to offer Ubuntu ESM a la carte without the other features of Advantage?</h4>
-      <p>Yes, in quantities of 1,000 machines or above, at $50/node/year. <br /><a href="/support/contact-us?product=server-esm">Contact Canonical Sales</a>.</p>
-
-      <h4 class="u-no-max-width">We're mirroring the repository on our internal Landscape Server. Is there a guide on how to get Ubuntu ESM if using Landscape?</h4>
+      <h4 class="u-no-max-width">We’re mirroring the repository on our internal Landscape Server. Can we still get Ubuntu ESM if using Landscape?</h4>
       <p>ESM is just a regular Ubuntu archive, but authenticated and served over HTTPS. Archive mirroring is already available in Landscape, and is the only supported mechanism for mirroring the ESM archive.</p>
 
       <h4 class="u-no-max-width">What will the LTS support situation look like during ESM? Will we be able to raise functionality tickets with Canonical or are you ending actual support and providing only security patches?</h4>
-      <p>The support window for Ubuntu 12.04 closes on April 28, 2017. The support team will not be able to fix bugs or build fixed packages once the 12.04 LTS archive is closed.</p>
+      <p>The support window for Ubuntu 12.04 LTS closes on April 28, 2017. The support team will not be able to fix bugs or build fixed packages once the 12.04 LTS archive is closed.</p>
```

Comment:
This "LTS" isn't in the copy doc, but I think it makes sense to include it here

Reply:
I updated the copy doc to reflect this.